package.json

```diff
@@ -43,14 +44,21 @@
     "deepmerge": "^4.2.1",
     "del": "^5.1.0",
     "font-awesome": "^4.7.0",
+    "headroom.js": "^0.11.0",
     "fs-extra": "^8.1.0",
     "merge-dirs": "^0.2.1",
     "minify": "^4.1.3",
     "node-sass": "^4.14.1",
     "npm-run-all": "^4.1.5",
     "postcss-cli": "^7.1.1",
+    "rollup": "^2.23.0",
+    "rollup-plugin-terser": "^6.1.0",
     "s72-kibble": "^0.10.24",
     "watch": "^1.0.2",
     "yargs": "^15.3.1"
+  },
+  "devDependencies": {
```

Comment:
From memory you won't need these if Headroom isn't there. They help resolve headroom.min.js in node_modules to an importable ES6 thing.